src/Algebra/FunctionProperties/Consequences.agda

```diff
@@ -164,6 +165,25 @@ comm+cancelʳ⇒cancelˡ {_•_} comm cancelʳ x {y} {z} eq = cancelʳ y z (begi
   x • z ≈⟨ comm x z ⟩
   z • x ∎)
 
+module InjectiveContravariantOperator {_•_ : Op₂ Carrier} {dual : Injection S S}
+         (let open Injection dual renaming (_⟨$⟩_ to _˘))
+         (contravariant : _˘ ContravariantOn _•_)         
+  where
```

Comment:
I doubt the whole `Injection` can be inferred from the function part.
I think `dual` should be an explicit argument.

Reply:
Without question!

I orginally has `dual` as a mere binary operation which could've been inferred from later frields, but then switched to `Injection` and forget to make it explicit.

Will make the change ^_^